frontend/src/employee-frontend/components/vasu/sections/VasuHeader.tsx

```diff
@@ -36,18 +37,26 @@ const Confidential = styled.div`
 `
 
 interface Props {
-  document: Pick<VasuDocument, 'basics' | 'documentState' | 'templateName'>
+  document: Pick<
+    VasuDocument,
+    'basics' | 'documentState' | 'templateName' | 'language'
+  >
 }
 export function VasuHeader({
   document: {
     basics: {
       child: { firstName, lastName }
     },
     documentState,
-    templateName
+    templateName,
+    language
   }
 }: Props) {
   const { i18n } = useTranslation()
+  const translations = vasuTranslations[
+    language.toLowerCase()
```

Comment:
Come to think of it, the language mapping function would be very nice because the vasu language might not always be a superset of UI languages, and the type casts are much harder to find when adding new languages.

Reply:
`function langToVasuLang(language: Lang): VasuLanguage` with explicit mapping and `fi` as fallback?

Reply:
Excellent point

Reply:
Reworked it a bit so that instead of having a mapping function, the `language` of the VasuDocument / VasuTemplate is directly the key to `vasuTranslations`